### PR TITLE
ceph: use jewel release

### PR DESCRIPTION
**Pull request by leseb**
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by alfredodeza** [comment]
The CentOS failure is valid:

    "msg": "Package at http://ceph.com/rpm-jewel/el7/noarch/ceph-release-1-0.el7.noarch.rpm could not be installed"

It is trying to get the ceph-release package from ceph.com (vs. download.ceph.com)

File: group_vars/all.sample [hunk 1]
```diff
@@ -82,8 +82,8 @@ dummy:
 # COMMUNITY VERSION
 #ceph_stable: false # use ceph stable branch
 #ceph_stable_key: https://download.ceph.com/keys/release.asc
-#ceph_stable_release: infernalis # ceph stable release
-#ceph_stable_repo: "http://ceph.com/debian-{{ ceph_stable_release }}"
+#ceph_stable_release: jewel # ceph stable release
+#ceph_stable_repo: "http://download.ceph.com/debian-{{ ceph_stable_release }}"
```
Comment (alfredodeza) [inline group_vars/all.sample]: line 100 would need to be updated to `download.ceph.com`

File: roles/ceph-common/defaults/main.yml [hunk 2]
```diff
@@ -74,8 +74,8 @@ ceph_use_distro_backports: false # DEBIAN ONLY
 # COMMUNITY VERSION
 ceph_stable: false # use ceph stable branch
 ceph_stable_key: https://download.ceph.com/keys/release.asc
-ceph_stable_release: infernalis # ceph stable release
-ceph_stable_repo: "http://ceph.com/debian-{{ ceph_stable_release }}"
+ceph_stable_release: jewel # ceph stable release
+ceph_stable_repo: "http://download.ceph.com/debian-{{ ceph_stable_release }}"
```
Comment (alfredodeza) [inline roles/ceph-common/defaults/main.yml]: below this in the stable releases section the URLs need to be updated as well

**Comment by alfredodeza** [comment]
Not sure why we are seeing this just today, but: `roles/ceph-common/tasks/installs/redhat_ceph_repository.yml` needs to update the urls from:

    http://ceph.com/rpm-{{ ceph_stable_release }}

To:

    http://download.ceph.com/rpm-{{ ceph_stable_release }}

**Comment by leseb** [comment]
@alfredodeza same here not sure why we are seing the url path issue only this time.
Will fix everything thanks!

**Comment by leseb** [comment]
test this please

**Comment by leseb** [comment]
test this please

**Comment by leseb** [comment]
test this please

**Comment by leseb** [comment]
test this please

**Pull request merged**